spec/Section 3 -- Type System.md

```diff
@@ -2113,3 +2119,94 @@ to the relevant IETF specification.
 ```graphql example
 scalar UUID @specifiedBy(url: "https://tools.ietf.org/html/rfc4122")
 ```
+
+### @defer
+
+```graphql
+directive @defer(
+  label: String
+  if: Boolean
```

Comment:
Given the other comments in this review, maybe this should be:

```suggestion
  if: Boolean! = true
```

Reply:
agree with @benjie both `@stream` and `@defer` shouldn't all `null` as a value for `if`.

Reply:
@benjie updated to change the type of `if` on both defer and stream to `Boolean! = true`